src/workflow/idle_workflow.c

```diff
@@ -48,36 +46,15 @@ static void _idle_workflow_init_communication(void)
     ui_screen_stack_push(info_centered_create("See the BitBoxApp", NULL));
 }
 
-static void _idle_workflow_spin(workflow_t* self)
+void idle_workflow_blocking(void)
 {
-    idle_workflow_data_t* data = self->data;
-    if (!data->initialized) {
-        data->delay_counter++;
-        if (data->delay_counter == IDLE_WORKFLOW_LOGO_DELAY) {
-            _idle_workflow_init_communication();
-            data->initialized = true;
-        }
-    }
-}
-
-static void _idle_workflow_init(workflow_t* self)
-{
-    idle_workflow_data_t* data = (idle_workflow_data_t*)self->data;
-    data->delay_counter = 0;
-    data->initialized = 0;
-}
-
-static void _idle_workflow_cleanup(workflow_t* self)
-{
-    (void)self;
-    ui_screen_stack_pop();
-}
-
-workflow_t* idle_workflow(void)
-{
-    return workflow_allocate(
-        _idle_workflow_init,
-        _idle_workflow_cleanup,
-        _idle_workflow_spin,
-        sizeof(idle_workflow_data_t));
+    component_t* waiting_screen = waiting_create();
+    UG_ClearBuffer();
+    waiting_screen->f->render(waiting_screen);
+    UG_SendBuffer();
+    waiting_screen->f->cleanup(waiting_screen);
+#ifndef TESTING
+    delay_ms(1300);
```

Comment:
this is manually tested to feel aoout the same duration as before (showing logo to switching to `See the BitBoxApp`).

the delay was originally added by fabian/djb as a UX/visual thing.